micro_http/src/common/headers.rs

```diff
@@ -13,39 +17,178 @@ pub enum Header {
     ContentLength,
     /// Header `Content-Type`.
     ContentType,
+    /// Header `Expect`.
+    Expect,
+    /// Header `Transfer-Encoding`.
+    TransferEncoding,
 }
 
 impl Header {
     fn raw(&self) -> &'static [u8] {
         match self {
             Header::ContentLength => b"Content-Length",
             Header::ContentType => b"Content-Type",
+            Header::Expect => b"Expect",
+            Header::TransferEncoding => b"Transfer-Encoding",
+        }
+    }
+
+    fn try_from(string: &[u8]) -> Result<Self, RequestError> {
```

Comment:
Since now we also parse HTTP headers, you should update the documentation in `micro_http/src/lib.rs` and add the supported ones.